datajunction-server/datajunction_server/models/node.py

```diff
@@ -505,6 +506,127 @@ def is_available(
         return True
 
 
+class MetricDirection(str, enum.Enum):
+    """
+    The direction of the metric that's considered good, i.e., higher is better
+    """
+
+    HIGHER_IS_BETTER = "higher_is_better"
+    LOWER_IS_BETTER = "lower_is_better"
+    NEUTRAL = "neutral"
+
+
+class MetricKind(str, enum.Enum):
+    """
+    The kind of metric
+    """
+
+    UNSPECIFIED = "unspecified"
+
+    COUNT = "count"
+
+    DELTA = "delta"
+
+    # A ratio is one quantity divided by another quantity, with the two sharing the same units
+    RATIO = "ratio"
+
+    # A rate is like ratio, but with different units of measure
+    RATE = "rate"
+
+    # A proportion is like ratio, but it compares a part to a whole.
+    # The possible values range from 0 to 1
+    PROPORTION = "proportion"
```

Comment:
This is not much different than ratio. In general I think we should start really small here and add new items when it is clear we need them.